README.md

```diff
@@ -160,6 +160,9 @@ import (
 )
 ```
 
+# Viewing job artifacts in openshift-ci
+Every Prow job executed by the CI system generates an artifacts directory containing information about that execution and its results. This [document](https://docs.ci.openshift.org/docs/how-tos/artifacts/) describes the contents of this directory and how they can be used to investigate the steps by the job.
+
```

Comment:
Can you please add here also the three most interesting (IMO) folders for us?
* appstudio-e2e-tests/redhat-appstudio-e2e/ - test logs
* appstudio-e2e-tests/redhat-appstudio-gather - pod logs, events, routes etc.
* appstudio-e2e-tests/redhat-appstudio-hypershift-gather - stonesoup, toolchain & tekton resources (like Applications, Components, Pipelineruns etc.)